### PR TITLE
Corrected Error Message

### PR DESCRIPTION
**Pull request by faze-geek**
Fixes a typo in `src/libasr/codegen/asr_to_wasm.cpp`

### CODE REVIEW

File: tests/reference/run_dbg-test_assert_01-2f34744.stderr [hunk 1]
```diff
@@ -1,5 +1,2 @@
-  File "tests/runtime_errors/test_assert_01.py", line 4
-    test()
-  File "tests/runtime_errors/test_assert_01.py", line 2
-    assert False
+warning: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```
Comment (Thirumalai-Shaktivel) [inline tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: Use `--skip-run-with-dbg`, while you update tests, so do `python run_tests.py -u --skip-run-with-dbg`. I forgot to document this in the README.

Comment (faze-geek) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: image

Comment (faze-geek) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: It isn't changing anything locally though.

Comment (Thirumalai-Shaktivel) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: Yup. This is correct as our tests do not test the above changes.

Comment (faze-geek) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: How can I correct the failing ubuntu and macos tests ? Manually changing back the `run-with-dbg` files ?

Comment (Thirumalai-Shaktivel) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: Undo the updated references and force push here.

Comment (faze-geek) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr]: My apologies for an incorrect attempt at reverting the initial commit. Now I have undone the updated references successfully.

**Review by czgdp1807 — APPROVED** [review]
Should be squashed and merged.

**Pull request merged**